test/e2e/monitoring/rules_test.go

```diff
@@ -32,32 +33,76 @@ var _ = Context("Prometheus Rules", func() {
 			CheckConfigCondition(gvk, ConditionAvailable, ConditionTrue, 15*time.Minute, CheckDoNotRepeat)
 		})
 
-		Context("CNAO rules", func() {
-			It("should have available runbook URLs", func() {
-				prometheusRule := monitoringv1.PrometheusRule{}
+		AfterEach(func() {
```

Comment:
moving the After each is ok by me, but please move it to a separate PR. this makes it hard to review the actual changes.

Reply:
Thanks, removed from this PR.